test/integration_test.js

```diff
@@ -779,10 +1059,10 @@ describe("Integration tests", () => {
             ]);
           });
 
-          it("should encode records when pushed to the server", () => {
```

Comment:
Are we still testing that it actually does do the encoding. The single biggest bug risk in terms of implications is if unencrypted data leaks out of a Kinto.js-based app due to some bug where the RemoteTransformer does not get applied in some edge case, so we should have supergood integration tests for that.